### ISSUE TITLE
RPC Integration test

### ISSUE THREAD

**Issue by adridadou** [adridadou/eth-contract-api#80]
Most of my tests / work is being done with EthereumJ.
I should have integration tests to make sure RPC works as well as the embeded version

**Comment by UrsZeidler** [comment]
For my tests of the smart contracts projects I use the [matrix job](https://wiki.jenkins-ci.org/display/JENKINS/Building+a+matrix+project) from Jenkins with the ethereum instances as axis. You just need to instantiate the ethereum facade by provided system properties.

**Comment by adridadou** [comment]
Thx for the info! I'll see what I can do with circleci. I guess that starting nodes shouldn't be too much of a problem. I already have scripts for geth.

**Comment by UrsZeidler** [comment]
Geth in the light mode is quite fast with syncing, but I think the Infura rcp is also a good choice.  (less infrastructure)

**Comment by adridadou** [comment]
yes not a bad idea. I was more thinking of a private network where I have control over who ether and who hasn't

**Comment by UrsZeidler** [comment]
Also quite interesting. Because this is the main "problem" provide enough ether and keep it this way. Something the build cannot provide by it self. Looking forward for it.
